packages/os/os.spec

```diff
@@ -315,15 +315,41 @@ migrations=()
 for migration in $(find %{_builddir}/sources/api/migration/migrations/v[0-9]* -mindepth 1 -maxdepth 1 -type d); do
     migrations+=("-p $(basename ${migration})")
 done
+
 # Store the output so we can print it after waiting for the backgrounded job.
+exec 3>&1 4>&2
```

Comment:
nit: a comment explaining the purpose (line 330) of fd 3 and fd 4 here would have helped me out. I experienced seconds and seconds of bemusement, tbh ("fd 3: what? Non-standard-wrong-error with bonus Hadamard matrix?"). Live and learn. This may just be me.

Reply:
> nit: a comment explaining the purpose (line 330) of fd 3 and fd 4 here would have helped me out.

Added a comment before the file descriptor juggling starts to explain what's going on.